pkgs/games/space-station-14-launcher/default.nix

```diff
@@ -0,0 +1,172 @@
+{ lib
+, buildDotnetModule
+, dotnetCorePackages
+, fetchFromGitHub
+, libX11
+, libICE
+, libSM
+, libXi
+, libXcursor
+, libXext
+, libXrandr
+, fontconfig
+, glew
+, SDL2
+, glfw
+, glibc
+, libGL
+, freetype
+, openal
+, fluidsynth
+, soundfont-fluid
+, gtk3
+, pango
+, atk
+, cairo
+, zlib
+, glib
+, gdk-pixbuf
+, wrapGAppsHook
+, buildFHSUserEnv
+, runCommand
+, iconConvTools
+, copyDesktopItems
+, makeDesktopItem
+}:
+
+let
+  rev = "0.20.5";
+  pname = "space-station-14-launcher";
+  space-station-14-launcher = buildDotnetModule rec {
+    inherit pname;
+
+    # Workaround to prevent buildDotnetModule from overriding assembly versions.
+    name = "${pname}-${rev}";
+
+    src = fetchFromGitHub {
+      owner = "space-wizards";
+      repo = "SS14.Launcher";
+      rev = "v${rev}";
+      hash = "sha256-uonndoqDOgPtnSk5v0KyyR8BQ9neAH1ploEY/kKD0IQ=";
+      fetchSubmodules = true;
+    };
+
+    buildType = "Release";
+    selfContainedBuild = false;
+
+    projectFile = [
+      "SS14.Loader/SS14.Loader.csproj"
+      "SS14.Launcher/SS14.Launcher.csproj"
+    ];
+
+    nugetDeps = ./deps.nix; # File generated with `nix-build -A space-station-14-launcher.passthru.fetch-deps`
```

Comment:
This should ideally be put into an update script

Reply:
I made an update script. However to make it work properly I had to made a separate file for the unwrapped launcher.